### PR TITLE
autofix-checks-lint: set fixed queries into Fix button

### PR DESCRIPTION
**Pull request by SergeyVinogorov**
Set fixed querie into link with warning message

### CODE REVIEW

**Comment by ai** [comment]
Please, resolve the conflicts. Without it, I can’t run preview deploy.

**Comment by SergeyVinogorov** [comment]
Here, i think, i missunderstood something, for check correct work new version of lint with fixed field
i did  pnpm install browserslist-lint@"here link to my repo with branch"
after that i got correct version with fixed key, and make changes for "Fix" button
Maybe i made something wrong?

**Comment by ai** [comment]
1. You also deleted `.tool-versions`
2. You need to rebase your branch

File: .tool-versions [hunk 1]
```diff
@@ -1,2 +1,2 @@
-nodejs 18.9.0
 pnpm 7.11.0
+nodejs 18.9.0
```
Comment (ai) [inline .tool-versions:2]: Keep file unchanged

File: client/view/QueryLink/QueryLink.css [hunk 2]
```diff
@@ -6,6 +6,11 @@
   &.is-pre {
     border-radius: 8px;
   }
+
+  &.fix {
```
Comment (ai) [inline client/view/QueryLink/QueryLink.css:10]: Modifier classes has `is-X` pattern

Comment (ai) [reply client/view/QueryLink/QueryLink.css:10]: Maybe `is-big` is a better name. We should explain what is different, not where it is used.

Comment (SergeyVinogorov) [reply client/view/QueryLink/QueryLink.css]: Maybe `is-block`? or you mean `is-big` like take more space?

Comment (ai) [reply client/view/QueryLink/QueryLink.css]: With `This branch has conflicts that must be resolved` I can’t see a preview and understand how this button looks like to recommend you a good name.

Comment (SergeyVinogorov) [reply client/view/QueryLink/QueryLink.css]: Pardon, resolved

Comment (ai) [reply client/view/QueryLink/QueryLink.css]: Oops, my script is not ready for third-party PR preview deploy :(.

Can you show me screenshot?

Comment (SergeyVinogorov) [reply client/view/QueryLink/QueryLink.css]: ok, few minutes

Comment (ai) [reply client/view/QueryLink/QueryLink.css]: Can you also rebase this commit? https://github.com/browserslist/browsersl.ist/commit/1e50f09395d855bbb6c0dce6ac37c79b26de946a

Comment (SergeyVinogorov) [reply client/view/QueryLink/QueryLink.css]: Screenshot 2022-09-25 at 14 29 41

Comment (ai) [reply client/view/QueryLink/QueryLink.css]: Looks bad:

1. Blue on yellow doesn’t fit. You need to change text color.
2. `display: block` tooks too many vertical space. It is better to move the button to the right of the warning.

Comment (SergeyVinogorov) [reply client/view/QueryLink/QueryLink.css]: i could leave here short video link if need?, and i think need to fix that when click on button, button is slide a little to the right, need to check what is this behavior

Comment (ai) [reply client/view/QueryLink/QueryLink.css]: 1. Let's rebase branch again. Maybe deploy will work this time and we could see live preview
2. It is better to fix position and color first. Maybe changes in position will fix `:active` behavior.

Comment (SergeyVinogorov) [reply client/view/QueryLink/QueryLink.css]: I have one question which color will be more compatible?, i tried orange, and marsianse that have on page, and both not expressed
Screenshot 2022-09-25 at 15 50 39
photo_2022-09-25 15 50 46

Comment (ai) [reply client/view/QueryLink/QueryLink.css]: Second option is a little better, but:
1. Text is orange but button background is transparent blue
2. I think it has low contrast


I see two option:
1. Use transparent black for background and black for text
2. Use darker orange for text and transparent background

File: client/view/Base/colors.css [hunk 3]
```diff
@@ -10,6 +10,7 @@
   --text-secondary: oklch(23% 0 0 / 65%);
   --text-underline: oklch(23% 0 0 / 40%);
   --text-underline-hover: oklch(23% 0 0);
+  --text-fix: oklch(72.5% 0.16 53.055);
```
Comment (ai) [inline client/view/Base/colors.css:13]: 1. `53.055` → `53`
2. Replace to some more generic name.

**Comment by ai** [comment]
Show screenshot

File: client/view/Alert/Alert.js [hunk 4]
```diff
@@ -12,9 +12,14 @@ export function buildError(root, message) {
   return error
 }
 
-export function buildWarning(root, message) {
+export function buildWarning(root, message, fixed) {
   let warning = createTag('div', ['Alert', 'is-warning'])
+  let fix = createTag('a', ['QueryLink', 'is-fix'])
```
Comment (ai) [inline client/view/Alert/Alert.js:17]: We need also to move button to the right side if the warning.

I suggest wrapping button to `div.Alert_action`

File: client/view/Base/colors.css [hunk 5]
```diff
@@ -10,6 +10,7 @@
   --text-secondary: oklch(23% 0 0 / 65%);
   --text-underline: oklch(23% 0 0 / 40%);
   --text-underline-hover: oklch(23% 0 0);
+  --text-dark-orange: oklch(72.5% 0.16 53);
```
Comment (ai) [inline client/view/Base/colors.css:13]: Still explain how it looks like, not the meaning. For instance, you can use `--text-warning`

File: server/package.json [hunk 6]
```diff
@@ -12,7 +12,7 @@
   },
   "dependencies": {
     "browserslist": "^4.21.4",
-    "browserslist-lint": "^0.2.1",
+    "browserslist-lint": "github:SergeyVinogorov/lint#autofix-checks-lint",
```
Comment (ai) [inline server/package.json:15]: You can use released version

File: client/view/Alert/Alert.js (hunk 4 above)
Comment (SergeyVinogorov) [reply client/view/Alert/Alert.js]: if I correctly understood, need to move button like on screenshot?
Screenshot 2022-09-29 at 13 21 12

Comment (ai) [reply client/view/Alert/Alert.js]: 1. No, we will have a lot of unused space in the top-right corner and bottom-left. Put the button to the top-right corner
2. Don't forget to check contrast in light and dark mode
3. In our design language all buttons have background. I don't see it on this screenshot.

**Comment by SergeyVinogorov** [comment]
Hello, sorry for taking so long to answer.
Can we use this button option?

Screenshot 2022-10-20 at 06 58 02
Screenshot 2022-10-20 at 06 59 01

**Comment by SergeyVinogorov** [comment]
I have not updated the branch yet, I want to clarify correctly, I caught the idea?
Screenshot 2022-10-20 at 11 58 52
Screenshot 2022-10-20 at 11 59 07

**Comment by SergeyVinogorov** [comment]
I checked contrast, in light mode 14+ 
I corrected contrast in dark mode setting in 13 + 
Is it will be ok, or left last version that I showed?
Screenshot 2022-10-20 at 12 43 58

**Comment by ai** [comment]
I like the latest version

**Comment by SergeyVinogorov** [comment]
ok I setup latest version with black background in black mode

**Pull request merged**